samples/bot-ai-meeting-helper/README.md

```diff
@@ -0,0 +1,186 @@
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+> When the application is running for the first time, the Teams toolkit will generate an app registration along with a password and other necessary credentials which is specified in "teamsapp.local.yml".
+
+## Setup and use the sample locally 
+1) Navigate to the `samples/bot-ai-meeting-helper/.localConfigs1` and rename to ".localConfigs" and update the values below.
+
+   ```txt
+        BOT_ID="BOT_ID"
+        BOT_PASSWORD="SECRET_BOT_PASSWORD"
+        AZURE_OPENAI_API_KEY="SECRET_AZURE_OPENAI_API_KEY"
+        AZURE_OPENAI_ENDPOINT="AZURE_OPENAI_ENDPOINT"
+        AZURE_OPENAI_DEPLOYMENT_NAME="AZURE_OPENAI_DEPLOYMENT_NAME" 
+        BOT_ENDPOINT="BOT_ENDPOINT"
+        Base64EncodedCertificate="Base_64_Encoded_Certificate"
+        EncryptionCertificateId="Encryption_Certificate_Id"
+        PRIVATE_KEY_PATH="Pem_File_Path"
+        Account_Name="Azure_Storage_Account"
+        Account_Key="Azure_Storage_Account_Key"
+        Table_Name="Azure_Storage_Table"
+        partitionKey="Azure_Storage_Table_PartitionKey"
+        AI_Model="Azure_Open_AI_Model"
+        SubscriptionURL="https://graph.microsoft.com/v1.0/subscriptions"
+        SystemPrompt="Generate a filtered list of action items from meeting transcriptions by user in bullet point user wise categorized with proper format like:  <b> Attendee:</b> 
+        Action Items in bullet points"
+        LocalTimeZone=Asia/Kolkata
+        APPINSIGHTS_INSTRUMENTATIONKEY=""
+        APPINSIGHTS_CONNECTIONSTRING=""
+    ``` 
+1) Create a policy for a demo tenant user for creating the online meeting on behalf of that user using the following PowerShell script (Images for reference only)
+ 
+      PowerShell script
+
+    ```powershell
+    # Import-Module MicrosoftTeams
+    # Call Connect-MicrosoftTeams using no parameters to open a window allowing for MFA accounts to authenticate
+    Connect-MicrosoftTeams
+    New-CsApplicationAccessPolicy -Identity “<<policy-identity/policy-name>>” -AppIds "<<microsoft-app-id>>" -Description "<<policy-description>>"
+    Grant-CsApplicationAccessPolicy -PolicyName “<<policy-identity/policy-name>>” -Identity "<<object-id-of-the-user-to-whom-the-policy-needs-to-be-granted>>"
+    
+    OR
+    # For global access
+    # Grant-CsApplicationAccessPolicy -PolicyName Meeting-policy-dev -Global
+    ```
+    Example:
+
+    ```powershell
+      # Import-Module MicrosoftTeams
+      Connect-MicrosoftTeams
+
+      New-CsApplicationAccessPolicy -Identity Meeting-policy-dev -AppIds "d0bdaa0f-8be2-4e85-9e0d-2e446676b88c" -Description "Online meeting policy - contoso town"
```

Comment:
Don't add the real APp id's here.

Reply:
This is not real App ID, but still changed